docs/man/kmscon.xml

```diff
@@ -289,6 +289,15 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--xkb-compose-file {file}</option></term>
+        <listitem>
+          <para>Path to a single predefined compose file. An empty path "" is
+                interpreted as using the default compose file. (default: load
+                the compose file corresponding to the current locale)</para>
```

Comment:
Please also mention that when no compose file is found, composing support is disabled.

Reply:
I've tried to describe the search algorithm in overview: https://github.com/Aetf/kmscon/pull/36/files#diff-5bbcc8fac2d9945e5e67a6b1d7f659da68428bfab4b8926c42dc90da8b1f1ebcR292-R308